octo-core/lib/octocore.rb

```diff
@@ -3,17 +3,22 @@
 require 'logger'
 
 require 'octocore/version'
-require 'octocore/utils'
 require 'octocore/config'
 require 'octocore/models'
 require 'octocore/counter'
+# require 'octocore/email'
```

Comment:
Can you try and fix the error? Maybe it's a gem dependency that is not mentioned in `octocore.gemspec` or maybe the spelling is wrong. The correct spelling is **Mandrill** iirc